frontend/src/app/integration.service.ts

```diff
@@ -115,8 +117,19 @@ export class IntegrationService {
     this.baseUrl = dataService.baseUrl;
   }
 
-  public startIntegration(schemaLeft: Schema, schemaRight: Schema) {
+  /**
+   *
+   * @param schemaLeft schema to be integrated that is referred to as left schema in UI
+   * @param schemaRight schema to be integrated that is referred to as right schema in UI
+   * @param thesaurus explanation: see server/definitions/ISchemaMatchingRequest.ts
+   */
+  public startIntegration(
+    schemaLeft: Schema,
+    schemaRight: Schema,
+    thesaurus?: string
+  ) {
     this._schemas = [schemaLeft, schemaRight];
+    this.thesaurus = thesaurus ?? this.thesaurus;
```

Comment:
was passiert hier? Ist Thesarus dann undefinden oder leer?
Wenn ich es richtig sehe wird es nicht vorher gesetzt, dann würde ich hier als default das reinschreiben, wass es sein soll. 
Oder kann evlt. noch eine File in der Variable sein?